docs/contributing/setting-up-your-local-dev-environment.md

```diff
@@ -26,7 +26,7 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 - Set up repo and install dependencies: `yarn run bootstrap`
 - Make sure tests are passing for you: `yarn test`
 - Create a topic branch: `git checkout -b topics/new-feature-name`
-- See [docs setup instructions](/contributing/docs-contributions#docs-site-setup-instructions) below for docs-only changes.
+- See [docs setup instructions](https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/docs-contributions.md#docs-site-setup-instructions) below for docs-only changes.
```

Comment:
This seems to be working for me without the change

Reply:
Yeah - let's keep this as is. Remember that our documentation is intended to be viewed on gatsbyjs.org, not in Github!

```suggestion
- See [docs setup instructions](/contributing/docs-contributions.md#docs-site-setup-instructions) below for docs-only changes.
```